pulsar-client-cpp/tests/AuthPluginTest.cc

```diff
@@ -412,6 +412,20 @@ TEST(AuthPluginTest, testOauth2RequestBody) {
     ASSERT_EQ(flow2.generateParamMap(), expectedResult2);
 }
 
+TEST(AuthPluginTest, getWellKnownOpenIdConfigurationUrl) {
+    std::string issuerUrl = "https://dev-kt-aa9ne.us.auth0.com";
```

Comment:
Nit: shouldn't we also test the case where there is a trailing `/` in the `issuerUrl`?

Reply:
+1

Reply:
Do you mean the case as flow2?

```
params["issuer_url"] = issuerUrl.append("/");
ClientCredentialFlow flow2(params);
ASSERT_EQ(flow2.getWellKnownOpenIdConfigurationUrl(), configurationUrl);
```

Reply:
Yes